lib/bno080/BNO080.cpp

```diff
@@ -1038,13 +1068,8 @@ void BNO080::softReset(void)
 	//Attempt to start communication with sensor
 	sendPacket(CHANNEL_EXECUTABLE, 1); //Transmit packet on channel 1, 1 byte
 
-	//Read all incoming data and flush it
-	delay(50);
-	while (receivePacket() == true)
-		; //delay(1);
-	delay(50);
-	while (receivePacket() == true)
-		; //delay(1);
+	waitForCompletedReset();
+	waitForCompletedReset();
```

Comment:
Do we need to do it twice after the change? I think it was doing this for both packets to arrive.

Reply:
The BNO resets twice on reset. So it sends both messages twice for a total of 4 messages. That said I do not want this merged yet without some extra testing from people who have the same hardware combination.

Reply:
I see, thank you.